pkg/rpc/wrappers/unspents.go

```diff
@@ -0,0 +1,56 @@
+package wrappers
+
+import (
+	"github.com/CityOfZion/neo-go/pkg/core"
+	"github.com/CityOfZion/neo-go/pkg/util"
+)
+
+// UnspentBalanceInfo wrapper is used to represent single unspent asset entry
+// in `getunspents` output.
+type UnspentBalanceInfo struct {
+	Unspents    []core.UnspentBalance `json:"unspent"`
+	AssetHash   util.Uint256          `json:"asset_hash"`
+	Asset       string                `json:"asset"`
+	AssetSymbol string                `json:"asset_symbol"`
+	Amount      util.Fixed8           `json:"amount"`
+}
+
+// Unspents wrapper is used to represent getunspents return result.
+type Unspents struct {
+	Balance []UnspentBalanceInfo `json:"balance"`
+	Address string               `json:"address"`
+}
+
+// GlobalAssets stores a map of asset IDs to user-friendly strings ("NEO"/"GAS").
+var GlobalAssets = map[string]string{
+	"c56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b": "NEO",
```

Comment:
why not to put them in config?

Reply:
They will be moved somewhere eventually, because more user-facing code needs this (though I'm really not sure that config is the right place). Theoretically, that's what `Name` field in `RegisterTX` and `AssetState` is for, but for these we have `AntShares` and `AntCoin` recorded in the blockchain for historical reasons, thus this map.